x-pack/plugins/apm/public/components/app/Settings/anomaly_detection/jobs_list.tsx

```diff
@@ -103,6 +176,30 @@ export function JobsList({ data, status, onAddEnvironments }: Props) {
             </h2>
           </EuiTitle>
         </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiSwitch
+            checked={showLegacyJobs}
+            onChange={(e) => {
+              setShowLegacyJobs(e.target.checked);
+            }}
+            label={i18n.translate(
+              'xpack.apm.settings.anomalyDetection.jobList.showLegacyJobsCheckboxText',
+              {
+                defaultMessage: 'Show legacy jobs',
+              }
+            )}
+          />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiButton href={mlManageJobsHref} color="primary">
+            {i18n.translate(
+              'xpack.apm.settings.anomalyDetection.jobList.manageMlJobsButtonText',
+              {
+                defaultMessage: 'Manage ML Jobs',
```

Comment:
```suggestion
                defaultMessage: 'Manage jobs',
```
Just want to get rid of the `ML` label